source/server-discovery-and-monitoring/tests/unified/minPoolSize-error.yml

```diff
@@ -0,0 +1,118 @@
+---
+description: minPoolSize-error
+
+schemaVersion: "1.10"
+
+runOnRequirements:
+    # failCommand appName requirements
+  - minServerVersion: "4.9"
+    topologies:
+      - single
```

Comment:
rather than using `directConnection=true`, I updated this test to only run against single topologies. That way test runners won't need to remove members of the seedlist.